kythe/extractors/gcp/README.md

```diff
@@ -19,7 +19,7 @@ export BUCKET_NAME="your-bucket-name"
 ## Hello World Test
 
 To make sure you have done setup correctly, we have an example binary at
-`kythe/go/extractors/gcp/helloworld`, which you can run as follows:
+`kythe/go/extractors/gcp/examples/helloworld`, which you can run as follows:
```

Comment:
`kythe/extractors/gcp/examples` ?

Reply:
ah yea.  That's what I get for fixing this link and then moving the file, instead of moving the file and then fixing the link.

Fixed some other bad pointers too